### PR TITLE
My Version of the answers

### PR DESCRIPTION
**Pull request by netikular**
I've got to implementations in the pipeline one!

### CODE REVIEW

File: lib/atm.ex [hunk 1]
```diff
@@ -18,7 +18,24 @@ defmodule ATM do
   Taken from codewars
   """
   def withdraw(n) do
-    # TODO : Code your solution
-    [0, 0, 0]
+    bills([0, 0, 0], n)
+  end
+
+  defguard test?(number, divisor) when number >= divisor and rem(number, divisor) == 0
```
Comment (ramortegui) [inline lib/atm.ex:24]: @netikular 🥇for you defguard!

Comment (netikular) [reply lib/atm.ex:24]: it was so hard to give it a name!! 

I finally went with a generic "test?" what would you call this guard?

Comment (ramortegui) [reply lib/atm.ex:24]: hmm it's hard!  well having the context, will something like:

defguard changeable?(amount, bill) when amount >= bill and rem(amount, bill) == 0
defguard dispensible?(amount, bill) when amount >= bill and rem(amount, bill) == 0

Comment (ramortegui) [reply lib/atm.ex:24]: or likely_changeble? lol

**Pull request merged**